src/modules/ekf2/EKF/estimator_interface.cpp

```diff
@@ -494,6 +494,17 @@ void EstimatorInterface::setDragData(const imuSample &imu)
 			}
 		}
 
+		// don't use any accel samples that are clipping
+		if (imu.delta_vel_clipping[0] || imu.delta_vel_clipping[1] || imu.delta_vel_clipping[2]) {
```

Comment:
so if the current accel sample is clipping, you reset the accumulator (containing the previous non-clipping values) and fill it with the current (clipped) sample?

Reply:
No, it returns before accumulating more.

Reply:
oh, my brain skipped the return